sway/input/cursor.c

```diff
@@ -369,7 +389,6 @@ static void handle_pointer_button(struct wl_listener *listener, void *data) {
 		}
 	}
 
-	cursor_handle_activity(cursor, IDLE_SOURCE_POINTER);
```

Comment:
Note for reviewer: I moved all calls to `cursor_handle_activity` to the start of functions, so figuring out if one ends up reporting an idle source change doesn't require scanning the whole function.